src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -576,64 +598,88 @@ private static unsafe void Ssse3Decode(ref byte* srcBytes, ref byte* destBytes,
                 8, 14, 13, 12,
                 -1, -1, -1, -1);
 
-            Vector128<sbyte> mask2F = Vector128.Create((sbyte)'/');
-            Vector128<sbyte> mergeConstant0 = Vector128.Create(0x01400140).AsSByte();
+            Vector128<byte> mask2F = Vector128.Create((byte)'/');
+            Vector128<byte> mergeConstant0 = Vector128.Create(0x01400140).AsByte();
             Vector128<short> mergeConstant1 = Vector128.Create(0x00011000).AsInt16();
-            Vector128<sbyte> zero = Vector128<sbyte>.Zero;
+            Vector128<byte> zero = Vector128<byte>.Zero;
```

Comment:
Based off latest head, but without any code changes... 

Zero is being created inside the loop every iteration:

```
IN003c: 0000F0  4F000419          movi    v25.4s, #0x00
IN003d: 0000F4  4E196AF7          trn2    v23.16b, v23.16b, v25.16b
```

I assume that's because I'm using Vector128<byte>.Zero instead of Create(0).

I'll tidy my code up to use Vector128<sbyte>.Zero inside the loop.